.github/workflows/ci.yml

```diff
@@ -10,26 +11,37 @@ on:
 
 jobs:
   main:
-    runs-on: ubuntu-latest
+    strategy:
+        matrix:
+          os: [ubuntu-latest]
+          build:
+              - java: 17
+                profile: codequality
+              - java: 11
```

Comment:
I'm not 100% we need to explicitly test Java 11 at the moment, since we are compiling on 8 and running all tests on 17. Between just those 2, we cover all our relevant checks. However, since we have a desire to make 11 the new minimum, we can keep this for now knowing eventually we will remove 8.